cibyl/plugins/__init__.py

```diff
@@ -46,15 +50,25 @@ def is_plugin_class(class_obj):
             "sources" in class_obj.__module__
 
 
-def extend_source(plugin_name, plugin_module_path):
-    plugin_sources = []
-    for py in [f[:-3] for f in os.listdir(plugin_module_path)
-               if f.endswith('.py') and f != '__init__.py']:
-        mod = __import__('.'.join(
-            [f"cibyl.plugins.{plugin_name}.sources", py]), fromlist=[py])
-        plugin_sources.extend([source_tuple[1] for source_tuple in
-                               inspect.getmembers(mod, is_plugin_class)])
```

Comment:
with this changes is_plugin_class is not called anywhere, can be safely removed

Reply:
Ok, will do so.